07-errors.md

```diff
@@ -0,0 +1,301 @@
+# Error Conditions
+
+This document specifies the error conditions in the FVM and the associated error codes/numbers.
+
+## Exit Codes
+
+Exit codes are numeric, 32bit integer codes that are returned when an actor exits both successfully and in error.
+Exit codes appear on-chain inside the message receipt, and therefore form part of consensus.
+
+We classify exit codes as "normal" and "abnormal".
```

Comment:
I personally don't think the _codes_ themselves are normal or abnormal; the execution is, resulting in success or failure exit codes. But I might be bikeshedding.

Reply:
Well, we classify exit codes into those that indicate "normal" versus "abnormal" exits, but I think the current wording better conveys the intent.